### PR TITLE
Alpha users lib

### PR DESCRIPTION
**Pull request by efertsch**

### CODE REVIEW

File: lib/conversation_handler.rb [hunk 1]
```diff
@@ -17,10 +17,10 @@ def converse
...
-      if @conversation.history.length == 1
-        return [CreateMessage.welcome, CreateMessage.triage_subscribable_type].join
+      if AlphaTesterValidator.phone_numbers.include?(@conversation.from)
+        return [CreateMessage.alpha_access_granted(@conversation.from), CreateMessage.triage_subscribable_type].join
       else
-        return "We already started, silly."
+        return CreateMessage.alpha_access_denied(@conversation.from)
       end
     else
       s_type = @conversation.get_subscribable_type if @conversation.has_subscribable_type?
```
Comment (jyonah) [inline lib/conversation_handler.rb]: It looks like this logic would only prevent someone from using the 'start' keyword, but the rest of the application would operate normally. Did you test against a user starting with "athlete" or a league name?

Comment (efertsch) [reply lib/conversation_handler.rb]: No, I'm aware of the edge cases. I figured it would be best to go over it with you and discuss the best course of action since you wrote the majority of this portion of the app.
